.github/pull_request_template.md

```diff
@@ -29,4 +29,4 @@ all targeted releases.
 
 -->
 
-1.14.7
+2.2.0
```

Comment:
Do we just want to remove this? I think this matters more when we have PRs coming in from lots of people that depend on landing in exact versions, but so far we just set milestone+`blocker` label for those that do.

Reply:
Fine by me :+1: